### DISCUSSION TITLE
创建集群 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1145] [Announcements]
# 创建集群 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/clusters/create-cluster/

**Comment by tony4gt** [comment]
1、创建集群的前提条件中要求K8S集群的版本要1.22+
2、产品截图中写的是1.21+
以上两者不一致，请确认哪个是正确的。

**Comment by yanrongshi** [comment]
感谢您的反馈，正确的创建集群的前提条件中要求K8S集群的版本应当是 1.22+ ，我们将进行修复
